src/pages/workspace/categories/ExpenseLimitTypeSelector/ExpenseLimitTypeSelector.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {useState} from 'react';
+import type {StyleProp, ViewStyle} from 'react-native';
+import {View} from 'react-native';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type {PolicyCategoryExpenseLimitType} from '@src/types/onyx/PolicyCategory';
+import ExpenseLimitTypeSelectorModal from './ExpenseLimitTypeSelectorModal';
+
+type ExpenseLimitTypeSelectorProps = {
+    /** Function to call when the user selects an expense limit type */
+    setNewExpenseLimitType: (value: PolicyCategoryExpenseLimitType) => void;
+
+    /** Currently selected expense limit type */
+    defaultValue: PolicyCategoryExpenseLimitType;
+
+    /** Label to display on field */
+    label: string;
+
+    /** Any additional styles to apply */
+    wrapperStyle: StyleProp<ViewStyle>;
+};
+
+function ExpenseLimitTypeSelector({defaultValue, wrapperStyle, label, setNewExpenseLimitType}: ExpenseLimitTypeSelectorProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const [isPickerVisible, setIsPickerVisible] = useState(false);
+
+    const showPickerModal = () => {
+        setIsPickerVisible(true);
+    };
+
+    const hidePickerModal = () => {
+        setIsPickerVisible(false);
+    };
+
+    const updateExpenseLimitTypeInput = (expenseLimitType: PolicyCategoryExpenseLimitType) => {
+        setNewExpenseLimitType(expenseLimitType);
+        hidePickerModal();
+    };
+
+    const title = translate(`workspace.rules.categoryRules.expenseLimitTypes.${defaultValue}`);
+    const descStyle = title.length === 0 ? styles.textNormal : null;
```

Comment:
why are you checking length? isn't it better to just evaluate title since empty string is still falsy?